service/controller/resource/snapshotid/error.go

```diff
@@ -0,0 +1,22 @@
+package snapshotid
+
+import (
+	"github.com/giantswarm/microerror"
+)
+
+var invalidConfigError = &microerror.Error{
+	Kind: "invalidConfigError",
+}
+
+var notExistsError = &microerror.Error{
+	Kind: "notExistsError",
+}
+
+var executionFailedError = &microerror.Error{
```

Comment:
Let's keep that on top with the comment. Check other packages how they do it.

Reply:
should look the same as tccp/error.